Runtime/org.codelite.codelite.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.codelite.codelite</id>
+  
+  <name>CodeLite</name>
+  <summary>CodeLite is an open source IDE for the C//C++ languages</summary>
```

Comment:
From Readme.md
> CodeLite is a free, open source, cross platform IDE specialized in C, C++, PHP and JavaScript (mainly for backend developers using Node.js) programming languages, which runs best on all major platforms (Windows, macOS and Linux).

Wonder if double `/` is intended

Reply:
Thanks for the feedback, I copied the summary from the Solus package. I will add the one from the Readme.md as it is better to be consistent.